### DISCUSSION TITLE
Does anyone know why load images/batch node keeps loading images out of sequence.

### DISCUSSION THREAD

**Discussion by Two82** [comfyanonymous/ComfyUI#2030] [Q&A] [answered]
It's very strange it keeps taking the first 20 or so images and dispersing them throughout the sequence every 11th frame. I have tried a few different nodes. I tried both in the vhs package and inspire package. I also tried deleting everything in my comfyui inputs folder and created a new folder and pasted images there, then copied path to nodes widget. Nothing seems to help. 

![Screenshot 2023-11-23 011857](https://github.com/WASasquatch/was-node-suite-comfyui/assets/126403375/3ca286ec-92f5-4196-ad51-e835cadfd60a)

 These I have circled should fall 3rd 4th 5th etc. in the sequence and I am confused why they're not. Thanks!

**Comment by ltdrdata** [comment, marked as answer]
I guess the filenames are... 

```
img1.png
img2.png
...
img9.png
img10.png
img11.png
```

In above case, the sorted order becoming...
```
img1.png
img10.png
img11.png
...
img2.png
img20.png
img21.png
...
img9.png
...
```
To avoid such case, you need to name it.
```
img001.png
img002.png
...
img009.png
img010.png
img011.png
...
```

**Reply by Two82** [reply]
Thank you, I should have known that. I have run into this issue several times before dealing with Pr and other like apps built for non-windows OS. Been error troubleshooting and didn't think about it. Was using video load but hate creating a video file just to split it in another program and lose quality.

**Reply by levicki** [reply]
Or the developers all over the world could finally start using [natural sort order](https://en.wikipedia.org/wiki/Natural_sort_order) which properly sorts text with numbers... One can dream that one day one will never see wrongly sorted filenames, right?

**Reply by RolfHihi** [reply]
![image](https://github.com/comfyanonymous/ComfyUI/assets/82800768/b9a43185-5d38-42af-b217-2e9d98a835e7)
Hmm I am using the correct notation but I still have the same issue. Or is the underscore somehow screwing it up

**Reply by ltdrdata** [reply]
That seems to be an entirely separate issue. It's necessary to verify first whether the your image loader supports the sort feature.

**Reply by RolfHihi** [reply]
Yeah you are right, I am using the Load Image Batch from Was Node suite. But it looked like a similar issue related to the default ordering method. I will try to contact the owner

**Reply by AllanChen** [reply]
Rename the image and use load images from path. that wolud be see the sequence images immediately. 

Vid2imgs
remember rename the image name as sequence, use this command
 `ffmpeg -i download.mp4 %4d.png`

After that set the image folder path into the load image node. like this.
![Screenshot 2024-04-16 at 14 19 23](https://github.com/comfyanonymous/ComfyUI/assets/1639667/f2fa9e4e-0d0d-42e5-b39c-60c6def12a81)
